apollo-backend/src/main/java/io/logz/apollo/controllers/DeploymentController.java

```diff
@@ -116,6 +116,8 @@ public void addDeployment(String environmentIdsCsv, String serviceIdsCsv, int de
             deploymentHandler.checkDeploymentShouldBeBlockedByRequestBlocker(serviceIds, environmentIds.size());
         } catch (ApolloDeploymentException e) {
             responseObject.addUnsuccessful(e);
+            assignJsonResponseToReq(req, HttpStatus.CREATED, responseObject);
```

Comment:
why CREATED? don't you want to return an error?

Reply:
@mikeskali Changed to Internal error

Reply:
Not internal error as well. Nothing is wrong internally. Bad request.

Reply:
@mikeskali Yeah sorry I mean bad request :).